samples/dsm/akeneo-dsm/back-end/app.js

```diff
@@ -22,6 +22,11 @@ app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
```

Comment:
I'm curious to know why do you have to set this header to * ?

Reply:
You're right, there's no need to wide open the app. It will now acccept only localhost:3000 request